support-workers/src/main/scala/com/gu/services/Services.scala

```diff
@@ -30,5 +32,6 @@ class Services(isTestUser: Boolean) {
   lazy val acquisitionService = AcquisitionServiceBuilder.build(isTestUser)
   lazy val promotionService = new PromotionService(promotionsConfigProvider.get(isTestUser))
   lazy val goCardlessService = new GoCardlessWorkersService(goCardlessConfigProvider.get(isTestUser))
+  lazy val redemptionService = RedemptionTable.forEnvAsync(TouchPointEnvironments.fromStage(stage, isTestUser))
```

Comment:
Can you use a config provider approach here similar to the 2 lines above for consistency?

Reply:
sure, I only didn't because it seemed unneccesary to have a class for one line of code, but will do

Reply:
it looks like the config provider system is designed to get the configuration from AWS store, but RedemptionTable is more like the AcquisitionServiceBuilder above in that it doesn't actually need any config, just needs to know the env.  So I propose to add the config provider as/when we need config.